packages/fluentui/react-northstar/src/components/Tree/hooks/useTree.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { TreeProps } from '../Tree';
+import { flattenTree, getTobeRenderedItemsProps } from './flattenTree';
+import { useTreeActiveState } from './useTreeActiveState';
+import { useTreeBehavior } from './useTreeBehavior';
+
+export function useTree(props: TreeProps) {
```

Comment:
I don't think that it's a right idea to depend on `TreeProps`, I suggest to create `UseTreeOptions` type and `TreeProps` can extend it.

Motivation: hook can be used without `Tree` component, `TreeProps` contains non related properties

```suggestion
export function useTree(options: UseTreeOptions) {
```